tests/unit/data/endpoints/rules/eventbridge.json

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
Were these generated internally or were they hand written? What's the difference between these files and the endpoint-tests.json files in the functional directory?

Reply:
these are considered 'vanilla' test cases. These were generated/written by the spec writers, not me. Vanilla in this case means not a real rule set belonging to an actual service. The rule sets that are used in functional tests are real rule sets